src/WebJobs.Extensions.DocumentDB/DocumentDBAttribute.cs

```diff
@@ -90,5 +91,13 @@ public DocumentDBAttribute(string databaseName, string collectionName)
         /// collection.
         /// </summary>
         public int CollectionThroughput { get; set; }
+
+        /// <summary>
+        /// Optional.
+        /// When specified on an input binding using an <see cref="IEnumerable{T}"/>, defines the query to run against the collection. 
```

Comment:
Yeah, that's the behavior I'd expect - just wanted to make sure we had the proper validations/user guidance in place